test/com/google/enterprise/adaptor/ad/AdAdaptorTest.java

```diff
@@ -1425,6 +1419,39 @@ public void testFakeAdaptorGetDocIds() throws Exception {
   }
 
   @Test
+  public void testMultipleFullGroupPushes() throws Exception {
```

Comment:
Add a TODO for a test of incremental pushes (e.g., full + incremental results in the union of groups).

Reply:
Done.